packages/clerk-js/src/core/clerk.ts

```diff
@@ -137,6 +144,14 @@ export default class Clerk implements ClerkInterface {
 
     this.proxyUrl = (options as ClerkConstructorOptions | undefined)?.proxyUrl;
 
+    if (this.proxyUrl) {
+      if (isValidProxyUrl(this.proxyUrl)) {
```

Comment:
:upside_down_face: Can't we have the `isValidProxyUrl` function handle undefined/null values and get rid of the double conditional here?

Reply:
I also dislike the nested conditionals here, but `proxyUrl` is an optional property, and I would like to throw an error if that property has set but it's format is wrong.
If I merge this logic, the the error will also throw if the proxyUrl is not set by the developer.